spec.md

```diff
@@ -555,7 +556,8 @@ If the request is invalid, such as a `<digest>` with an invalid syntax, a `400 B
 
 Upon success, the response MUST be a JSON body with an image index containing a list of descriptors.
 Each descriptor is of an image manifest in the same `<name>` namespace with a `subject` field that specifies the value of `<digest>`.
-The descriptors MUST include an `artifactType` field that is set to the value the configuration descriptor's `mediaType` for an image manifest.
+The descriptors MUST include an `artifactType` field that is set to the value of the `artifactType` in the image manifest.
```

Comment:
This part doesn't seem to be consistent with the image spec proposal. In the image spec the 'artifactType' is only mandatory if the config descriptor `mediaType` is scratch.

So if an image has a non-scratch config descriptor `mediaType` and an `artifactType`, it makes sense for anyone reading the manifest to ignore the `artifactType` altogether, including the registry implementation. 

Or else the registry may return an `artifactType` matching the `artifactType`, and after the artifact is copied locally, another tool with a different implementation may only look at the config descriptor. 

That is unless we make sure the `artifactType` is absent from any image having a non-scratch config descriptor `mediaType`. Meaning the image spec mandates `artifactType` and non-scratch config descriptor `mediaTypes` are mutually exclusive.